Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM maven:3.6.3-adoptopenjdk-8 AS build
+
+WORKDIR /usr/src/app
+COPY pom.xml .
+RUN mvn -B -e -C -T 1C org.apache.maven.plugins:maven-dependency-plugin:3.1.1:go-offline
+
+COPY src src
+RUN mvn -B -e -o -T 1C package -DskipTests
+
+FROM tomcat:9-jdk8-adoptopenjdk-hotspot AS publish
```

Comment:
Can you add the procedures to build and publish? I think you need to run Elasticsearch before publishing.

Reply:
The procedures are written in PR overview.
This Dockerfile is only used for building `personium-core` container image. It is not related to running.

To launch personium Unit with this container image, you should launch other containers like below docker-compose.yml.

```yml
services:
  elasticsearch:
    image: elasticsearch:6.8.12
  activemq:
    image: rmohr/activemq:5.15.9
  memcached-lock:
    image: memcached:1.6.7
  memcached-cache:
    image: memcached:1.6.7
  nginx:
    build:
      context: ./dockerfiles/nginx
  personium-engine:
    build:
      context: ./dockerfiles/personium-engine
  personium-core:
    image: personium-core:1.7.22
```

This PR enables you to write personium-core settings like this.

```yml
  personium-core:
    image: personium-core:1.7.22
```

Reply:
Okay, I see. Then, how about addding the following procedures to `README.md`?

----
## Build Docker image

### Tomcat including personium-core.war

```
docker build . -t personium-core
```

### Building environment (maven)

```
docker build . --target build -t personium-core-build
```